packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1446,6 +1446,16 @@ enum FocusHighlightStrategy {
   alwaysTraditional,
 }
 
+// By extending the WidgetsBindingObserver class,
+// we can add a listener object to FocusManager as a private member.
+class _AppLifecycleListener extends WidgetsBindingObserver {
+  _AppLifecycleListener(this.onLifecycleStateChanged);
+  final void Function(AppLifecycleState) onLifecycleStateChanged;
```

Comment:
Is Flutter's style to have an empty line between the constructor and member?

```suggestion
  _AppLifecycleListener(this.onLifecycleStateChanged);

  final void Function(AppLifecycleState) onLifecycleStateChanged;
```